src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -94,14 +101,14 @@ You can locate elements by their `class`, `id`, `link text`, `name` or even XPat
     id="locate-by-link-text"
     title="Locate by link text"
   >
-    Locate an element by its displayed link text; for example, `<a href="http://example.com>your link text here</a>`. You can locate by the <DoNotTranslate>**exact**</DoNotTranslate> link text (`$driver.By.linkText`) or by partial match (`$driver.By.partialLinkText`):
+    Locate an element by its displayed link text; for example, `<a href="http://example.com>your link text here</a>`. You can locate by the <DoNotTranslate>**exact**</DoNotTranslate> link text (`$selenium.By.linkText`) or by partial match (`$selenium.By.partialLinkText`):
 
     ```
-    $browser.findElement($driver.By.linkText("Configuration Panel"));
+    await $webDriver.findElement($selenium.By.linkText("Configuration Panel"));
     ```
 
     ```
-    $browser.findElement($driver.By.partialLinkText("Configuration Pa"));
+    await $webDriver.findElement($selenium.By.partialLinkText("Configuration Pa"));
     ```
 
     For an example of locating a link by its link text, see [Navigate to a link](#locate-by-link-text).
```

Comment:
Links back to this section. Should this be removed?

Reply:
Good catch! I've removed it.